io.github.lephotographelibre.BlueNotebook.yaml

```diff
@@ -0,0 +1,114 @@
+app-id: io.github.lephotographelibre.BlueNotebook
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-23.08' 
+command: bluenotebook
+finish-args:
+  # Accès au serveur d'affichage (X11 et Wayland)
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Requis pour X11 (mémoire partagée)
+  - --share=ipc
+  # Accès à l'accélération graphique (pour WebEngine)
+  - --device=dri
+  # Accès au réseau (Météo, YouTube, etc.)
+  - --share=network
+  # Accès au son (Vidéos YouTube)
+  - --socket=pulseaudio
+  # Accès aux fichiers utilisateur (pour sauvegarder le journal)
+  # On peut restreindre à xdg-documents si on veut être plus strict
+    # --filesystem=xdg-documents
+  - --filesystem=home
+  # Accès au fichier de configuration sur l'hôte (au lieu du dossier privé de la sandbox)
+  #- --filesystem=xdg-config/BlueNotebook
+  # Accès au portail du bureau pour une meilleure intégration (ouverture de liens)
+  #- --talk-name=org.freedesktop.portal.Desktop
+  # Permet de lire la liste des applications par défaut pour éviter le sélecteur
+  #- --filesystem=xdg-config/mimeapps.list:ro
+  # Forcer fontconfig à lire la configuration de la police emoji
+  - --env=FONTCONFIG_FILE=/app/etc/fonts/fonts.conf
+
+modules:
+  # 1. Dépendances Python générées précédemment
+  - python3-requirements.yaml
+
+  # Ajout de la police Noto Color Emoji pour un affichage correct des emojis
+  - name: noto-color-emoji
+    buildsystem: simple
+    build-commands:
+      - install -d -m 755 /app/share/fonts/
+      - install -m 644 NotoColorEmoji.ttf /app/share/fonts/
+      - install -d -m 755 /app/etc/fonts/conf.d/
+      - install -m 644 75-noto-color-emoji.conf /app/etc/fonts/conf.d/
+      # Installer le fichier maître fonts.conf
+      - install -m 644 fonts.conf /app/etc/fonts/fonts.conf
+      - fc-cache -f -v /app/share/fonts/
+    sources:
+      - type: file
+        url: https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf
+        sha256: 72a635cb3d2f3524c51620cdde406b217204e8a6a06c6a096ff8ed4b5fd6e27b
+      - type: file
+        path: 75-noto-color-emoji.conf
+      - type: file
+        path: fonts.conf
+
+  # Ajout de la police Noto Sans
+  - name: noto-sans
+    buildsystem: simple
+    build-commands:
+      - install -d -m 755 /app/share/fonts/
+      - install -m 644 NotoSans-*.ttf /app/share/fonts/
+      - fc-cache -f -v /app/share/fonts/
+    sources:
+      - type: file
+        url: https://github.com/notofonts/noto-fonts/raw/main/hinted/ttf/NotoSans/NotoSans-Regular.ttf
```

Comment:
URL should NEVER reference the tip of a branch. It's not immutable.

And same below.